python/ray/_private/runtime_env/context.py

```diff
@@ -22,13 +22,17 @@ def __init__(
         command_prefix: List[str] = None,
         env_vars: Dict[str, str] = None,
         py_executable: Optional[str] = None,
+        executable: Optional[str] = None,
+        args: List[str] = None,
         resources_dir: Optional[str] = None,
         container: Dict[str, Any] = None,
         java_jars: List[str] = None,
     ):
         self.command_prefix = command_prefix or []
         self.env_vars = env_vars or {}
-        self.py_executable = py_executable or sys.executable
+        # TODO(omus): deprecate `py_executable` in favor of `executable`
```

Comment:
I'm not sure this is a good idea; it's possible that users may want to mix julia + python in the same runtime, so I think they need to be able to specify them separately.  I don't think we want to make users jump through additional hoops with per-task runtime environments just to be able to mix julia + python tasks in a single job.

Reply:
Oh, are you thinking that the same runtime environment is being used to launch workers for various languages? That's is a scenario I wasn't considering. I'll give that some thought

Reply:
I kept the Julia stuff separate to ensure we don't run into problems here.